code/ship/ship.cpp

```diff
@@ -4331,6 +4331,8 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 				ci->texture_stretch = 1.0f;
 			}
 		}
+		else
+			ci->texture_stretch = 1.0f;
```

Comment:
This is not quite correct.  This branch will always be hit when parsing a modular table that doesn't have the `+Bitmap Stretch:` tag, even if the tag was in a previously parsed table.

Changing this to `else if (first_time)` should be sufficient.

Reply:
Ah, right, right, thank you